controllers/volumepool_controller.go

```diff
@@ -81,6 +81,12 @@ func (r *VolumePoolReconciler) reconcileExists(ctx context.Context, log logr.Log
 	if !pool.DeletionTimestamp.IsZero() {
 		return r.delete(ctx, log, pool)
 	}
+
+	if pool.Name != r.VolumePoolName {
+		log.V(1).Info("Skipped pool, since it was not created by controller", "pool", client.ObjectKeyFromObject(pool))
```

Comment:
```suggestion
		log.V(1).Info("Skipping VolumePool, since it is not owned by us")
```

Reply:
No need to print the pool as it is automatically done by the `VolumePoolReconciler` logger. (the main `For` type is always printed in the structured log)